### PR TITLE
added functionality to wishlist

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
https://user-images.githubusercontent.com/71331989/161267748-93c73d69-f2b5-4681-b006-a3ea265f2402.mp4

### CODE REVIEW

**Review by rdshinde — APPROVED** [review]
I think it is good to merge.

File: src/components/wishlist/wishlist.jsx [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
           />
         </div>
          <div class="flex-row space-between">
         
-        <div className="card-header-one">Camera</div>
+        <div className="card-header-one">{item.name}</div>
         <div class="rating-comp flex-row display-flex-center">
-        <span class="star-icon">4
+        <span class="star-icon">{item.rating}
         <FaHeart/>
```
Comment (raman1999) [inline src/components/wishlist/wishlist.jsx:28]: you can destructure item  to get its properties

**Review by raman1999 — APPROVED** [review]
looks good

File: src/components/wishlist/wishlist.jsx (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply src/components/wishlist/wishlist.jsx:28]: okay...will do it

**Pull request merged**